siteorigin-panels.php

```diff
@@ -135,13 +136,19 @@ public static function is_legacy_browser(){
 	 * @param $class
 	 */
 	public static function autoloader( $class ) {
-		if ( strpos( $class, 'SiteOrigin_Panels_' ) === 0 ) {
-			$filename = strtolower( str_replace( array( 'SiteOrigin_Panels_', '_' ), array( '', '-' ), $class ) );
+		$filename = false;
+		if ( strpos( $class, 'SiteOrigin_Panels_Widgets' ) === 0 ) {
+			$filename = str_replace( 'SiteOrigin_Panels_Widgets_', '', $class );
+			$filename = preg_replace( '/([a-z])([A-Z])/', '$1-$2', $filename );
+			$filename = plugin_dir_path( __FILE__ ) . 'inc/widgets/' . strtolower( $filename ) . '.php';
```

Comment:
Ideally, yeah... but we can't rename widget classes after they're out in the wild. That would break content.